main.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+
+using namespace std;
+
+int main() {
+    //Constants for Joe's Purchasing of the stocks.
+    double joes_buying_price = 45.50;
+    int joe_shares_bought = 1000;
+    double stockbroker_commission = .02;
+
+    //First round of Math
+    double joe_paid = joes_buying_price * joe_shares_bought;
+    double commission_paid_buying = joe_paid * stockbroker_commission;
+
+    //Constants for Joe's Selling of Stocks
+    double joes_selling_price = 56.90;
```

Comment:
Looks good. Only suggestion format the constants at the top of page.
const double  SELLING_PRICE = 56.90 for example